crates/loader/src/local/mod.rs

```diff
@@ -125,26 +130,38 @@ pub fn validate_raw_app_manifest(raw: &RawAppManifestAnyVersion) -> Result<()> {
     Ok(())
 }
 
+fn filter_components(components: &Vec<OsString>, c: &RawComponentManifest) -> bool {
```

Comment:
Suggest `include_component`.  `filter` implies a set operation, such as filtering the list of RawComponentManifests to just the included ones..